src/Control/Effect/Lift.hs

```diff
@@ -3,6 +3,11 @@
 {- | Provides a mechanism to kick off the evaluation of an effect stack that takes place in a monadic context.
 
 'Lift' effects are always the last effect in a given effect stack. These stacks are invoked with 'Control.Effect.Lift.runM'. The 'Control.Effect.Pure.Pure' effect is equivalent to @Lift Identity@.
+
+Predefined carriers:
+
+* "Control.Carrier.Lift"
+* 'IO'
```

Comment:
This is awesome.

Reply:
Should we give an example somewhere of using `runM` versus discharging to `IO` directly?

Reply:
Maybe, but I don’t think we need to hold up 1.0 for it.